controllers/nodefeaturediscovery_controller.go

```diff
@@ -479,21 +455,6 @@ func (r *NodeFeatureDiscoveryReconciler) SetupWithManager(mgr ctrl.Manager) erro
 		Complete(r)
 }
 
-// validateUpdateEvent validates whether or not an NFD Operator resource has
-// an update.
-func validateUpdateEvent(e *event.UpdateEvent) bool {
```

Comment:
Can you add to the PR description an explanation for deleting this function?

Reply:
Done